src/first.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Returns the first element to be returned by an Observable
+ *
+ * @param {Observable} input Input Observable
+ * @return {Observable} New Observable
+ */
+
+const first = input => {
+  let out;
+  input.subscribe(x => {
+    if (!out) out = x;
+  });
+  return out;
+};
+
+first._name = 'first';
+
+export default first;
```

Comment:
We'll need it to propagate errors and complete messages as well. What if it errors or completes before the first one?